tests/test_browser.go

```diff
@@ -251,7 +251,7 @@ func (b *testBrowser) asGojaBool(v interface{}) bool {
 }
 
 // runString in the goja runtime.
-func (b *testBrowser) runString(s string, args ...interface{}) (goja.Value, error) {
+func (b *testBrowser) runString(s string, args ...interface{}) (goja.Value, error) { //nolint:unparam
```

Comment:
Is there no way to avoid the `nolint:unparam`?

Reply:
I can disable the `wrapcheck` linter for `test_browser` and `k6test`. Then I can avoid id. I'll do that in a minute.